firebase-common/src/main/java/com/google/firebase/platforminfo/PlatformInfoToken.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.platforminfo;
+
+/** The class is not public to ensure other components cannot depend on it. */
+class PlatformInfoToken {
+  final String key;
+  final String value;
+
+  PlatformInfoToken(String key, String value) {
```

Comment:
I feel like this needs some sort of namespacing per library for it to be PlatformInfo?
Is this intended to be just version or are we planning to start capturing other things in here?
```
PlatformInfoToken(String sdk, String key, String value) {
}
```

Reply:
There are no plans to capture anything other than sdk/library versions, so I feel like (key,value) pairs are sufficient to represent that?